internal/controller/instaslice_daemonset.go

```diff
@@ -708,74 +699,69 @@ func (r *InstaSliceDaemonsetReconciler) updateNodeCapacity(ctx context.Context,
 			log.FromContext(ctx).Error(err, "failed to marshal original node")
 			return err
 		}
-		if allocationStatus == string(inferencev1alpha1.AllocationStatusCreating) {
+		if allocation.Allocationstatus == inferencev1alpha1.AllocationStatusCreating {
 			// assume only one quantity can be requested for a profile
-			resourceName := "nvidia.com/mig-" + profile
+			resourceName := "nvidia.com/mig-" + allocation.Profile
 			// Count InstaSlice extended resources in Capacity
 			countCapacity := 0
 			for resourceName := range node.Status.Capacity {
 				if strings.HasPrefix(string(resourceName), orgInstaslicePrefix) {
 					countCapacity++
 				}
 			}
+			log.FromContext(ctx).Info("capacity while creating", "count", countCapacity)
 			resourceQuantity := resource.NewQuantity(1, resource.DecimalSI)
-			extendedResQuantity := resource.NewQuantity(int64(countCapacity), resource.DecimalSI)
+			extendedResCapacityQuantity := resource.NewQuantity(int64(countCapacity), resource.DecimalSI)
 			existingCapacity, capacityExists := node.Status.Capacity[v1.ResourceName(resourceName)]
-			existingAllocatable, allocatableExists := node.Status.Allocatable[v1.ResourceName(resourceName)]
 			if capacityExists {
-				if extendedResQuantity.Cmp(existingCapacity) == 1 {
+				if existingCapacity.Cmp(*extendedResCapacityQuantity) == -1 {
+					log.FromContext(ctx).Info("Adding resource ", "name", resourceName, "pod", allocation.PodName)
```

Comment:
We should probably discuss logging in a separate issue.

This to me should not be a general informational log as this could get quite noisy.

Reply:
I agree, can you please open an issue?